src/nwb2bids/_messages/_inspection_message.py

```diff
@@ -0,0 +1,15 @@
+import pydantic
+
+
+class InspectionMessage(pydantic.BaseModel):
```

Comment:
Idea: include an optional 'examples' field that showcases what the value should look like